volatility3/framework/plugins/mac/pslist.py

```diff
@@ -105,10 +107,20 @@ def _generator(self):
             self.config["kernel"],
             filter_func=self.create_pid_filter(self.config.get("pid", None)),
         ):
+            offset = format_hints.Hex(task.vol.offset)
+            name = utility.array_to_string(task.p_comm)
             pid = task.p_pid
+            uid = task.p_uid
+            gid = task.p_gid
+            start_time_seconds = task.p_start.tv_sec
+            start_time_microseconds = task.p_start.tv_usec
+            start_time = datetime.datetime.fromtimestamp(
+                start_time_seconds + start_time_microseconds / 1e6
+            )
```

Comment:
Why don't you create a new associated function on the `task` object to use that information later on? These [examples](https://github.com/volatilityfoundation/volatility3/blob/2149231bb7db3c7fc48e589a16124cca373da331/volatility3/framework/symbols/windows/extensions/__init__.py#L741) can be found in `windows.pslist`.

Reply:
And these changes will need change the `_required_framework_version`.